pandas/types/concat.py

```diff
@@ -201,6 +201,57 @@ def convert_categorical(x):
         return Categorical(concatted, rawcats)
 
 
+def union_categoricals(to_union):
```

Comment:
Add a `ignore_order=False` kwarg? That would prevent changed/copied orig categoricals if you ever need this... Would only disable the order check

```py
if not ignore_order and any(c.ordered for c in to_union):
        raise TypeError("Can only combine unordered Categoricals")
```

It would still return a unordered cat, of course.